src/components/stockExchangeData.js

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { IoIosArrowBack } from 'react-icons/io';
+import { ShowStockData } from '../redux/Reducer';
+
+function StockExchangeData() {
+  const stocks = useSelector((state) => state.stocks);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(ShowStockData());
+  }, []);
+  const Navigate = useNavigate();
+
+  return (
+
+    <div className="chosen_data">
+
+      <button type="button" className="">
+        {' '}
+        <IoIosArrowBack onClick={() => { Navigate('/home'); }} />
+      </button>
+      <div className="symbol_name">
+        {stocks.symbol ? (
+          <h4>
+            {stocks.symbol}
+
+          </h4>
+        ) : 'Chosen Item'}
+
+      </div>
+
+      <div className="detailsDiv">
+
+        <p>
+          {' '}
+          Ask price(usd) :
+          {' '}
+          <span>{stocks.askPrice}</span>
+        </p>
+        {' '}
+        <p>
+          {' '}
+          Bid price(usd) :
+          {' '}
+          <span>{stocks.bidPrice}</span>
```

Comment:
- [ ] it would be better to make sure the header of the detail page is responsive on all mobile screen in order to match the design
- yours 
<img width="401" alt="Screenshot 2022-12-01 at 12 09 02" src="https://user-images.githubusercontent.com/61790587/205038040-ad7d3c0d-3b39-4a54-b244-42432b231b88.png">

Reply:
add media queries